packages/zarf-registry/zarf.yaml

```diff
@@ -58,6 +58,40 @@ variables:
     default: ""
     autoIndent: true
 
+  - name: REGISTRY_CREATE_SERVICE_ACCOUNT
+    description: Toggle the ability to use an IRSA for the registry
+    default: "false"
+
+  - name: REGISTRY_SERVICE_ACCOUNT_NAME
+    description: The name of the service account to use. If not set and create is true, a name is generated using fullname template
+    default: ""
+
+  - name: REGISTRY_SERVICE_ACCOUNT_ANNOTATIONS
+    description: The ARN of the IAM Role to assume
```

Comment:
Recommend taking out explicit mention of AWS things here since it should be general

Reply:
New descriptions are more generic/appropriate